tester_app/client/package.json

```diff
@@ -0,0 +1,77 @@
+{
+  "name": "react-webpack-rails-tutorial",
+  "version": "1.1.0",
+  "description": "Code from the React Webpack tutorial.",
+  "main": "server.js",
+  "engines": {
+    "node": "4.2.0",
+    "npm": "3.3.6"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/shakacode/react-webpack-rails-tutorial.git"
+  },
+  "keywords": [
+    "react",
+    "tutorial",
+    "comment",
+    "example"
+  ],
+  "author": "justin808",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/shakacode/react-webpack-rails-tutorial/issues"
+  },
+  "homepage": "https://github.com/shakacode/react-webpack-rails-tutorial",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start": "node server.js",
+    "build:client": "NODE_ENV=production webpack --config webpack.client.rails.config.js",
+    "build:server": "NODE_ENV=production webpack --config webpack.server.rails.config.js",
+    "build:dev:client": "webpack -w --config webpack.client.rails.config.js",
+    "build:dev:server": "webpack -w --config webpack.server.rails.config.js",
+    "lint": "npm run eslint && npm run jscs",
+    "eslint": "eslint --ext .js,.jsx .",
+    "jscs": "jscs --verbose ."
+  },
+  "dependencies": {
+    "babel-core": "^5.8.25",
+    "babel-loader": "^5.3.2",
+    "body-parser": "^1.14.1",
+    "es5-shim": "^4.1.14",
+    "es6-promise": "^3.0.2",
+    "expose-loader": "^0.7.0",
+    "immutable": "^3.7.5",
+    "imports-loader": "^0.6.4",
+    "jquery": "^2.1.4",
+    "jquery-ujs": "^1.1.0-1",
+    "loader-utils": "^0.2.11",
+    "marked": "^0.3.5",
+    "react": "^0.14.0",
+    "react-bootstrap": "^0.27.0",
```

Comment:
currently can't disable bootstrap anyway, but will note this for if and when we do so in the future